### PR TITLE
customize scheduler interval.

### PR DESCRIPTION
**Pull request by morvencao**
This PR tries to support customize scheduler interval for moving policy compliance to history, can be 'month', 'week', 'day', 'hour', 'minute' or 'second'.

### CODE REVIEW

File: doc/simulation/local-policies/README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ tmpfs            64M     0   64M   0% /dev
 tmpfs            31G     0   31G   0% /sys/fs/cgroup
 tmpfs            31G   84M   31G   1% /etc/passwd
 /dev/nvme0n1p4  120G   30G   90G  26% /tmp
-/dev/nvme4n1     20G    3G   20G   2% /pgdata
+/dev/nvme4n1     20G  1.3G   20G   2% /pgdata
```
Comment (yanmxa) [inline doc/simulation/local-policies/README.md:93]: @morvencao Did you intend to update the test document for the local policy here, or did you add it to this PR carelessly?

Comment (morvencao) [reply doc/simulation/local-policies/README.md:93]: I want to update it in this PR, since it's small change for the storage usage.

Comment (yanmxa) [reply doc/simulation/local-policies/README.md:93]: okay

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**